src/index.html

```diff
@@ -12,11 +12,50 @@
     />
     <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./style.css"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="/">
+          <img
+            src="images/logo.png"
+            alt="Логотип компанії MOYO"
```

Comment:
The `alt` attribute should describe the image content. The current content seems to be in a different language (Ukrainian) than the specified language of the page (English). If the website is intended for an English-speaking audience, the `alt` attribute should be in English.